lib/gpu_cluster_status.rb

```diff
@@ -113,11 +113,19 @@ def gpus_percent
       ((total_gpus - full_nodes_available).to_f / total_gpus.to_f) * 100
     end
 
-    # Return the queued GPUs as percent of available GPUs
+    # Return the percentage of queued jobs requesting gpus
     #
     # @return [Float] The percentage GPUs queued
-    def gpus_queued_percent
-      (queued_gpus.to_f / total_gpus.to_f) * 100
+    def queued_gpu_jobs_percent(available_jobs)

```

Comment:
a better name for this method would be: `percent_of_queued_jobs_requesting_gpus`